src/cargo/core/features.rs

```diff
@@ -364,6 +364,7 @@ pub struct CliUnstable {
     pub weak_dep_features: bool,
     pub extra_link_arg: bool,
     pub credential_process: bool,
+    pub future_incompat_report: bool,
```

Comment:
I've gotten confused by how many different occurrences of "future incompat report" is used in various places. I think this should be renamed `enable_future_incompat_reporting` since this is just what enables the lints in rustc.

Reply:
I think `enable_future_incompat_feature` would be better, to emphasize that this doesn't necessarily enable displaying the report. I'll rename some of the other duplicate field names as well.